src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DrawerAdapter.java

```diff
@@ -0,0 +1,38 @@
+package edu.kis.powp.jobs2d.drivers.adapter;
+
+import edu.kis.legacy.drawer.panel.DrawPanelController;
+import edu.kis.legacy.drawer.shape.ILine;
+import edu.kis.legacy.drawer.shape.LineFactory;
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.features.DrawerFeature;
+
+/**
+ * driver adapter to drawer with several bugs.
+ */
+public class DrawerAdapter extends DrawPanelController implements Job2dDriver {
+    private int startX = 0, startY = 0;
```

Comment:
Original code used tabs instead of spaces. It is important to keep the original convention. It makes tracking actual changes more difficult.